coder_sniffer/Drupal/Sniffs/Arrays/ArraySniff.php

```diff
@@ -109,23 +111,26 @@ public function process(File $phpcsFile, $stackPtr)
         }
 
         if ($isInlineArray === true) {
-            // Check if this array contains at least 3 elements and exceeds the 80
-            // character line length.
-            if ($tokens[$tokens[$stackPtr][$parenthesisCloser]]['column'] > $this->lineLimit) {
+            // Check if this array has more than one element and exceeds the
+            // line length defined by $this->lineLimit.
+            $currentLine = $tokens[$stackPtr]['line'];
+            $tokenCount  = $stackPtr;
+            while ($tokenCount < ($phpcsFile->numTokens - 1) && $tokens[($tokenCount + 1)]['line'] === $currentLine) {
+                $tokenCount++;
```

Comment:
why do we need to count here? We already know the array is on a single line. We can use the column from $parenthesisCloser.

Reply:
See https://github.com/pfrenssen/coder/pull/114#issuecomment-652974349 at 2 Jul 13:24 BST above. The `$parenthesisCloser` is often not the last thing on the line, there will often be a comma or semi-colon, but there may be more and we cannot assume anything. So this count gets all the content on that line to make sure we get the accurate length.